CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## [0.8.0] - 2024-10-03
+
+### Breaking
+- `Webhook#update` now takes an Object of `{notificationUri, eventTypeFilter}` instead of a string of notification Uri. 
```

Comment:
`Webhook#update` now takes an Object of `{notificationUri, eventTypeFilter}` instead of `notificationUri` string

Reply:
modified.